utils/build/docker/python/flask/exception_replay_controller.py

```diff
@@ -0,0 +1,43 @@
+from flask import Blueprint, request, abort
+from debugger.exception_replay import ExceptionReplayPaper, ExceptionReplayRock, ExceptionReplayScissors
+
+exception_replay_blueprint = Blueprint("exceptionreplay", __name__, url_prefix="/exceptionreplay")
+
+
+@exception_replay_blueprint.route("/simple", methods=["GET"])
```

Comment:
Are those endpoint documented in `docs/weblog` ?

Reply:
Yes:
https://github.com/DataDog/system-tests/tree/main/docs/weblog#get-exceptionreplay